### ISSUE TITLE
Caltech is correct, not "Cal Tech"

### ISSUE THREAD

**Issue by danielskatz** [BloombergMedia/whatiscode#110]
Just a note about an incorrect abbreviation in the article - see subject line for all the details

**Comment by yuningalexliu** [comment]
Good catch @danielskatz, I addressed this in https://github.com/BloombergMedia/whatiscode/pull/118.
